browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -8176,12 +8183,15 @@ index 592fa4c4d9a45eb1e9b95e0cdabc8d404b40018d..50b310adfdea35e6194120f5b776632d
  }
  
  DownloadProxy::~DownloadProxy()
-@@ -178,7 +183,18 @@ void DownloadProxy::decideDestinationWithSuggestedFilenameAsync(DownloadID downl
+@@ -178,7 +183,21 @@ void DownloadProxy::decideDestinationWithSuggestedFilenameAsync(DownloadID downl
  {
      if (!m_processPool)
          return;
 -    
 +
++    if (auto* instrumentation = m_dataStore->downloadInstrumentation())
```

Comment:
Rather than adding a new event for the suggested name is it possible to move downloadCreated instrumentation over here?

Reply:
It is not!

Reply:
Can you elaborate? AFAICT the instrumentation could defer sending the messages until suggested file name is available. The frontend anyways is going to wait for it.